backend/ethereum/channel/conclude.go

```diff
@@ -28,7 +28,7 @@ import (
 	"perun.network/go-perun/channel"
 )
 
-const secondaryWaitBlocks = 2
+var secondaryWaitBlocks = int(TxFinalityDepth + 2)
```

Comment:
I just realized that my previous suggestion is problematic because `TxFinalityDepth` might change.

What do you think about keeping `const secondaryWaitBlocks = 2` here and when we use it in `conclude.go` L65, we write `waitConcludedForNBlocks(waitCtx, a, events, TxFinalityDepth + secondaryWaitBlocks)`?